tests/0117-mock_errors.c

```diff
@@ -255,6 +255,58 @@ static void do_test_offset_commit_request_timed_out(rd_bool_t auto_commit) {
         SUB_TEST_PASS();
 }
 
+/**
+ * @brief A COORDINATOR_LOAD_IN_PROGRESS error is returned from
```

Comment:
Sorry but this might make sense now, but when reading this in the future it will seem as if we're expecting consumer_poll() to return COORDINATOR_LOAD_IN_PROGRESS. So maybe rephrase as:
"Verify that a cluster roll does not cause consumer_poll() to return the temporary and retriable COORDINATOR_LOAD_IN_PROGRESS error. We should backoff and retry in that case."

Reply:
Makes sense, especially since there's nothing to refer to besides this comment. I changed it to be like what you have recommended.